ports/management/commands/fetch-build-history.py

```diff
@@ -1,11 +1,11 @@
 from django.core.management.base import BaseCommand, CommandError
 
-from parsing_scripts import parse_build_history
```

Comment:
I think `parse_build_history.py` can be removed now unless you have kept it for some reasons?